### DISCUSSION TITLE
Multiple copies of Skpy running

### DISCUSSION THREAD

**Discussion by boboliiii** [Terrance/SkPy#269] [Q&A] [answered]
Hi all,

I looked for an answer to this and couldn't find anything (although I'm not sure how other people would word the same question)-

If I run two copies of Skpy at the same time, with different credentials, they both hit Auth Rate Limit Exceeded at the same times..   Even if the instances are run at different times and have had different activity (and are running from different IP addresses).

Any ideas how to avoid these being associated together?  Like I said, it's different login credentials and different IPs..   Should I make a physical second copy of Skpy?  Or is this just Auth Rate Limit for anyone using Skpy?  

Any advice or ideas welcome!
Thanks

**Comment by Terrance** [comment, marked as answer]
Skype may have recently started enforcing tighter limits (see #267), so if your two copies have comparable usage then they may be independently hitting the same new limit.

Otherwise, I can only speculate that they've become entangled another way and are now sharing a rate limit.  (e.g. Have you _ever_ signed into both accounts from the same IP?)
